src/Cipher/Vigenere/VigenereStream.tsx

```diff
@@ -7,6 +7,10 @@ class VigenereStream extends KeyedCipherStreamBase {
   constructor(props: Props) {
```

Comment:
Can we reuse the Props class from KeyedCipherBase instead of redeclaring it here?  I forgot to play around with that.